core/equations.gms

```diff
@@ -576,38 +576,35 @@ q_emiTeMkt(t,regi,emiTe(enty),emiMkt) ..
       vm_emiTeDetailMkt(t,regi,enty2,enty3,te,enty,emiMkt)
     )
     !! energy emissions fuel extraction
-  + v_emiEnFuelEx(t,regi,enty)$(sameas(emiMkt,"ETS"))
+  + v_emiEnFuelEx(t,regi,enty)$( sameas(emiMkt,"ETS") )
     !! Industry CCS emissions
-	- sum(emiInd37_fuel,
-		  vm_emiIndCCS(t,regi,emiInd37_fuel)
-		)$( sameas(enty,"co2") AND sameas(emiMkt,"ETS"))
-    !! substract carbon from biogenic or synthetic origin contained in
-    !! plastics that don't get incinerated ("plastic removals")
-  - sum(entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt),
-      sum(se2fe(entySe,entyFe,te)$( entySeBio(entySe) OR entySeSyn(entySe) ),
-        vm_nonIncineratedPlastics(t,regi,entySe,entyFe,emiMkt)
-      )
+  - sum(emiInd37_fuel,
+      vm_emiIndCCS(t,regi,emiInd37_fuel)
+    )$( sameas(enty,"co2") AND sameas(emiMkt,"ETS") )
+    !! substract carbon from non-fossil origin contained in plastics that don't
+    !! get incinerated ("plastic removals")
+  - sum((entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt),
+         se2fe(entySe,entyFe,te))$( entySeNonFos(entySe) ),
+      vm_nonIncineratedPlastics(t,regi,entySe,entyFe,emiMkt)
     )$( sameas(enty,"co2") )
-    !! add emissions from plastics incineration. CHECK FOR DOUBLE-COUNTING RISK
-  + sum(entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt),
-      sum(se2fe(entySe,entyFe,te),
-        vm_incinerationEmi(t,regi,entySe,entyFe,emiMkt)
-      )
+    !! add fossil emissions from plastics incineration. 
+  + sum((entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt),
+         se2fe(entySe,entyFe,te))$( entySeFos(entySe) ),
+      vm_incinerationEmi(t,regi,entySe,entyFe,emiMkt)
     )$( sameas(enty,"co2") )
-    !! add emissions from chemical feedstock with unknown fate
-  + sum(entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt),
-      sum(se2fe(entySe,entyFe,te),
-        vm_feedstockEmiUnknownFate(t,regi,entySe,entyFe,emiMkt)
-      )
+    !! add fossil emissions from chemical feedstock with unknown fate
+  + sum((entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt),
+         se2fe(entySe,entyFe,te))$( entySeFos(entySe) ),
```

Comment:
This is the only algorithmic change, the rest is refactoring, right?

Reply:
Yes, everything in the "fix chemical feedstock emissions from non-fossil sources" commit.